vaadin-crud-flow-parent/vaadin-crud-flow-integration-tests/src/test/java/com/vaadin/flow/component/crud/tests/CompositeIT.java

```diff
@@ -28,8 +28,8 @@ public void compositeTouchesDirtyState() {
         $(TextFieldElement.class).attribute("editor-role", "language-field")
                 .first().setValue("English");
 
-        $(TestBenchElement.class).id("overlay").$(ButtonElement.class).first()
-                .click();
+        $(TestBenchElement.class).withId("overlay").last()
```

Comment:
i read this from the TB release note
```
Differently from the previous versions, ElementQuery.id(...) will now throw an exception if multiple elements with the same id are present, instead of returning the first matching item. To switch back to the old behavior, you have to change the id(...) call to withId(...).first().
```

but here we use `withId(...).last()` ?